pyaedt/modules/Boundary.py

```diff
@@ -354,6 +368,334 @@ class NativeComponentPCB(NativeComponentObject, object):
 
     def __init__(self, app, component_type, component_name, props):
         NativeComponentObject.__init__(self, app, component_type, component_name, props)
+        self._filter_map2name = {"Cap": "Capacitors", "Ind": "Inductors", "Res": "Resistors"}
+
+    @property
+    @pyaedt_function_handler()
+    def footprint_filter(self):
+        """Minimum component footprint for filtering."""
+        if self._app.settings.aedt_version < "2024.2":
+            return None
+        return self.filters.get("FootPrint", {}).get("Value", None)
+
+    @footprint_filter.setter
+    @pyaedt_function_handler()
+    @disable_auto_update
+    def footprint_filter(self, minimum_footprint):
+        """Set minimum component footprint for filtering.
+
+        Parameters
+        ----------
+        minimum_footprint : str
+            Value with unit of the minimum component footprint for filtering.
+        """
+        self.auto_update = False
+        if self._app.settings.aedt_version < "2024.2":
+            return False
+        new_filters = self.props["NativeComponentDefinitionProvider"].get("Filters", [])
+        if "FootPrint" in new_filters:
+            new_filters.remove("FootPrint")
+        if minimum_footprint is not None:
+            new_filters.append("FootPrint")
+            self.props["NativeComponentDefinitionProvider"]["FootPrint"] = minimum_footprint
+        self.props["NativeComponentDefinitionProvider"]["Filters"] = new_filters
+
+    @property
+    @pyaedt_function_handler()
+    def power_filter(self):
+        """Minimum component power for filtering."""
+        return self.filters.get("Power", {}).get("Value", None)
```

Comment:
```suggestion
        return self.filters.get("Power", {}).get("Value")
```
dict `get` returns `None` by default if the key is not found